pkg/project/secrets_test.go

```diff
@@ -77,6 +77,7 @@ func (suite *SecretsExpanderTestSuite) BeforeTest(suiteName, testName string) {
 	suite.graphMock.ProjectByOrgAndName(mock.NoOptions)
 
 	suite.cfg, err = config.New()
+	suite.cfg.Set(constants.UnstableConfig, true)
```

Comment:
Since this isn't mocking the config running this test would mess with your own local config. A simple config mock might be better, eg. http://github.com/ActiveState/cli/blob/e8c10335bc8ee70a5e7f01d4f91b55b7c472cef1/internal/updater/updater_test.go#L34-L34

Reply:
Thanks for pointing this out. I had no idea.